freqtrade/resolvers/strategy_resolver.py

```diff
@@ -123,7 +121,8 @@ def _load_strategy(
         current_path = Path(__file__).parent.parent.joinpath('strategy').resolve()
 
         abs_paths = [
-            config['user_data_dir'].joinpath('strategies'),
+            Path(config['user_data_dir']).joinpath('strategies'),
+            config['strategy_path'],
```

Comment:
This line is killing CI ... strategy_path is not guaranteed to be available (only if you specify `--strategy-path xxx`, so this line will almost always fail.

It's also a duplicate entry (so it's searching this location twice) since `extra_dir` will contain exactly this path if it's available.